src/parameters/all_parameters.cpp

```diff
@@ -222,6 +228,7 @@ void AllParameters::parse_parameters (dealii::ParameterHandler &prm)
     else if (test_string == "shock_1d")                                 { test_type = shock_1d; }
     else if (test_string == "reduced_order")                            { test_type = reduced_order; }
     else if (test_string == "POD_adaptation")                           { test_type = POD_adaptation; }
+    else if (test_string == "finite_difference_sensitivity")            { test_type = finite_difference_sensitivity; }
```

Comment:
Consider moving this to a unit test similarly to:
https://github.com/dougshidong/PHiLiP/tree/master/tests/unit_tests/sensitivities
Also, the name finite-difference-sensitivity is a bit generic.

Reply:
I would rather keep it as an integration test since I use this to generate my snapshots. If it's a unit test, I would have to manually copy the files from the unit tests directory to the integration tests directory so I can use them for a POD, which would be pretty inconvenient.

Is the name `finite_difference_state_sensitivity_wrt_parameter` ok? Or more simply`fd_state_sensitivity_wrt_parameter`?

Reply:
That's fine for the testing, just checking that it's what is required.

Yeah, the renaming to `fd_state_sensitivity_wrt_parameter`, is already less generic.